backend/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "backend",
```

Comment:
maybe this name is to generic...

Reply:
I agree - changed to products-backend-server

Reply:
I agree, changed to products-backend-server.